install/infra/terraform/aks/kubernetes.tf

```diff
@@ -20,37 +15,37 @@ resource "azurerm_role_assignment" "k8s_reader" {
 }
 
 resource "azurerm_kubernetes_cluster" "k8s" {
-  name                = format(local.name_format, local.location, "primary")
-  location            = azurerm_resource_group.gitpod.location
-  resource_group_name = azurerm_resource_group.gitpod.name
-  dns_prefix          = "gitpod"
-  tags                = {}
-  api_server_authorized_ip_ranges     = []
-
-  kubernetes_version               = data.azurerm_kubernetes_service_versions.k8s.latest_version
+  name                            = format(local.name_format, local.location, "primary")
+  location                        = azurerm_resource_group.gitpod.location
+  resource_group_name             = azurerm_resource_group.gitpod.name
+  dns_prefix                      = "gitpod"
+  tags                            = {}
+  api_server_authorized_ip_ranges = []
+
+  kubernetes_version               = var.cluster_version
```

Comment:
I have a feeling that this may break in time. Azure tend to remove versions of k8s at fairly short notice, which is why the `data "azurerm_kubernetes_service_versions"` is the recommended way of getting the versions. You can use a [version_prefix](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/kubernetes_service_versions#version_prefix) as well

My suggestion is to keep this change as you've done it, but do so in the knowledge that we may have to change it in the future.